src/class-fields/initializer-error/cls-decl-fields-comp-name.template

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2017 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: static-semantics-early-errors
```

Comment:
sec-class-definitions-static-semantics-early-errors

This should be the final id for this section.